### PR TITLE
fixed splitImage function, if registry and port are present, but no tag

### PR DESCRIPTION
**Pull request by pgrund**

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -8,10 +8,10 @@ package main
 import (
   _ "github.com/gliderlabs/logspout/httpstream"
   _ "github.com/gliderlabs/logspout/routesapi"
-  _ "github.com/rtoma/logspout-redis-logstash"
+  _ "github.com/BisnodeInformatics/logspout-redis-logstash"
```
Comment (rtoma) [inline build.sh]: I can't merge this of course :-)

File: build.sh [hunk 2]
```diff
@@ -8,10 +8,10 @@ package main
...
 )
 MODULES
 
-docker build -t rtoma/logspout-redis-logstash -f Dockerfile.example .
+docker build -t bisnode/logspout-redis-logstash -f Dockerfile.example .
```
Comment (rtoma) [inline build.sh]: Same here.

**Comment by rtoma** [comment]
Thanks for this pull request!
Please revert the bisnode / BisnodeInformatics changes and I will gladly merge.

File: build.sh (hunk 2 above)
Comment (pgrund) [inline build.sh]: sorry .. committed too fast :-D

**Comment by pgrund** [comment]
reverted :-D

**Comment by rtoma** [comment]
Thanks, Peter.

I was about to merge when a thought occurred to me: does a missing tag equal 'latest'?
What does `docker images` tell you? If it shows 'latest' we could use that in the code for your case.

**Comment by pgrund** [comment]
yes indeed .. missing tag implies 'latest'
I'll have a look into it next week ...

**Pull request merged**